Dockerfile

```diff
@@ -19,7 +19,7 @@ FROM base as Development
 
 
 # Development Specific needs here
-CMD ["poetry", "run", "flask", "run", "--host",  "0.0.0.0."]    
+CMD ["poetry", "run", "flask", "run", "--host",  "0.0.0.0"]    
```

Comment:
Only thing remaining - you've got the same typo with an extra `.` in the `0.0.0.0` for your Gunicorn command below as well